### ISSUE TITLE
Error building module named `entry`

### ISSUE THREAD

**Issue by nexcvon** [Nuitka/Nuitka#1218]
```bash
$ echo "#" > entry.py
nuitka --module entry.py   # fail

$ mv entry.py entry2.py
nuitka --module entry2.py   # success

$ nuitka --version
0.6.9.0
Python: 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)]
Executable: python.exe
OS: Windows
Arch: x86_64
```

**Comment by kayhayen** [comment]
Rather old Nuitka you have there, but the reproducer still works on latest one. I have pushed a fix to factory, a variable name `module_entry` was using for the internal module table entry as well as for the module name. For me, it compiled with warnings, but it surely wouldn't work well in all cases. Solved by renaming that variable to not clash into the module names namespace.

**Comment by kayhayen** [comment]
Released as part of 0.6.17 just now.
